bin/spice/cmd/trace.go

```diff
@@ -26,23 +26,33 @@ import (
 )
 
 var (
-	// If `--last` was requested
-	isLast bool
-
 	// The id of the trace to provide
 	id string
 
 	// The trace_id of the trace to provide
 	trace_id string
 )
 
+var supported_trace_tasks = []string{"ai_chat", "ai_completion", "sql_query", "nsql"}
```

Comment:
This does not cover `tool_use`

`tool_use::sample_data`
`tool_use::table_schema`